persist/persist_test.go

```diff
@@ -1,9 +1,39 @@
-package persist
+package persist_test
```

Comment:
why change pkg name?

Also no Apache license header

Reply:
to avoid an import cycle